.env.example

```diff
@@ -1,3 +1,27 @@
+## CHANIN SUPPORT
+
```

Comment:
@jamiehewitt15 I think you added an env var (`FILECOIN_PRODUCTION`) for production or testing environment (depending how it's set). I personally don't want to change the default env vars names that are already in use.

Reply:
Yeah exactly